### PR TITLE
Skipping ARM64 builds temporarily

### PR DESCRIPTION
**Pull request by MariusNi**
## Description

Disabling the ARM64 builds for now.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-osconfig/blob/main/CONTRIBUTING.md).
- [x] All unit tests are passing.
- [x] I have merged the latest `main` branch prior to this PR submission.
- [x] I submitted this PR against the `main` branch.

### CODE REVIEW

File: devops/README.md [hunk 1]
```diff
@@ -21,16 +21,16 @@ Amazon Linux 2 | x86_64 | [insiders-fast](#amazonlinux2-insiders-fast)
 Azure Linux 2 (mariner-2) | x86_64 | [insiders-fast](#azurelinux2-insiders-fast)
 CentOS 7 | x86_64 | [insiders-fast](#centos7-insiders-fast)
 CentOS 8 | x86_64 | [insiders-fast](#centos8-insiders-fast)
-Debian 10 | x86_64 / arm64 | [insiders-fast](#debian10-insiders-fast) / [prod](#debian10-prod)
-Debian 11 | x86_64 / arm64 | [insiders-fast](#debian11-insiders-fast) / [prod](#debian11-prod)
-Debian 12 | x86_64 / arm64 | [insiders-fast](#debian12-insiders-fast) / [prod](#debian12-prod)
+Debian 10 | x86_64 | [insiders-fast](#debian10-insiders-fast) / [prod](#debian10-prod)
```
Comment (AhmedBM) [inline devops/README.md:24]: Were still previously releasing those packages and we will be shortly anyways. I wouldn't update the README given we will re-introduce once our rate-limits expire.

Comment (MariusNi) [reply devops/README.md:24]: We will re-add then. Until when we will readd publishing of ARM64 packages, we should not falsely advertise for them. Even for older ones that exist - and we do not want people to use unless they really know what they are doing, in which case documentation won't matter.  Do you agrere?

Comment (MariusNi) [reply devops/README.md]: Reverted these. We still have the part with "If you see any missing platforms or architectures that would help, please contact osconfigcore@microsoft.com."

**Review by AhmedBM — APPROVED** [review]
👌

**Pull request merged**